mypy/nodes.py

```diff
@@ -619,13 +620,14 @@ def __init__(self,
         super().__init__()
         self.arguments = arguments
         self.arg_names = [arg.variable.name for arg in self.arguments]
-        self.arg_kinds: List[int] = [arg.kind for arg in self.arguments]
-        self.max_pos = self.arg_kinds.count(ARG_POS) + self.arg_kinds.count(ARG_OPT)
-        self.body = body
+        self.arg_kinds: List[ArgKind] = [arg.kind for arg in self.arguments]
+        self.max_pos: int = (
+            self.arg_kinds.count(ARG_POS) + self.arg_kinds.count(ARG_OPT))
+        self.body: 'Block' = body
         self.type = typ
         self.unanalyzed_type = typ
         self.is_overload = False
-        self.is_generator = False
+        self.is_generator: bool = False
```

Comment:
Why add the type to only this field? Just curious

Reply:
Because something about making some of the things ArgKind's prompted `__init__` to get deferred, I'd guess, which meant that the fields weren't all inferred properly, and this is one of the ones that got caught up in a problem with one of the use sites. I annotated some more to make it less weird, but sigh.